homeassistant/components/knx/climate.py

```diff
@@ -57,7 +58,8 @@
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
     vol.Required(CONF_TEMPERATURE_ADDRESS): cv.string,
-    vol.Required(CONF_TARGET_TEMPERATURE_ADDRESS): cv.string,
+    vol.Required(CONF_TARGET_TEMPERATURE_STATE_ADDRESS): cv.string,
+    vol.Optional(CONF_TARGET_TEMPERATURE_ADDRESS): cv.string,
```

Comment:
Requires a default value?

Reply:
CONF_TARGET_TEMPERATURE_ADDRESS != CONF_TEMPERATURE_ADDRESS

Reply:
The value used to be required, now it is optional, it is usually a good policy to add a default value to optional parameters

Reply:
Every user uses different group addresses. A default value doesn't make sense here - also, not all users will need this value anymore thus changing it to optional